### DISCUSSION TITLE
Consulta parcial Basher

### DISCUSSION THREAD

**Discussion by ezecastiglione18** [dds-jv/foro#333] [Consultas de objetos] [answered]
Buenas! Estaba viendo el parcial de Basher y me trabe con una parte. Cuando se habla de marco legal, yo ahi no entiendo si hay que implementar ese marco en una Interfaz o como una clase concreta mas y que cada clausula externa herede de ese marco. Si alguien pudo hacerlo ese parcial me ayudaria mucho! Saludos

**Comment by gastonprieto** [comment, marked as answer]
hola @ezecastiglione18 

el marco seria una plantilla de un contrato y tendria un conjunto de clausulas, q c/clausula sabe generar el texto q corresponde al contrato
